### PR TITLE
#168244900 Implements user can edit comment on a trip request

### PR DESCRIPTION
**Pull request by Abobos**
**What does this PR do?**

Implements user can edit a comment on a trip request

**Description of Task to be completed?**

- Create a PUT route to api/v1/comment/trips/:commentUuid
- Create a  method in comment controller to handle the logic to update a comment

**How should this be manually tested?**

* Open terminal
* Clone this repo
* run ``npm install``
* run ``npm run dev``
* Open Postman
* Goto PUT http://localhost:3000/api/v1/comment/trips/:commentUuid
* Enter your request details like so:

  * message: 'Please approve my request'
  

**Any background context you want to provide?**

None

**What are the relevant stories?**
[#168244900](https://www.pivotaltracker.com/story/show/168244900)

**Screenshots (if appropriate)**

Postman 9_6_2019 8_19_53 AM

**Questions:**

None

### CODE REVIEW

**Review by Wokoro — APPROVED** [review]
LGTM.

**Review by dieudonneAwa — APPROVED** [review]

**Review by richienabuk — APPROVED** [review]

**Review by justinefe — APPROVED** [review]

**Review by justinefe — APPROVED** [review]

File: src/controllers/CommentController.js [hunk 1]
```diff
@@ -21,19 +21,21 @@ class CommentController {
  */
   static async createTripRequestComment(req, res, next) {
     const checkrequestOwner = async (tripRequestOwner, userUuid, userRole) => {
-      if (userRole === 'Manager') {
+      // checking if the user is a manager
```
Comment (chukwuemekachm) [inline src/controllers/CommentController.js]: No need for these comments.

File: src/controllers/TripRequestController.js [hunk 2]
```diff
@@ -51,6 +51,7 @@ class TripRequestController {
       if (!tripDeparture) return sendErrorResponse(res, 404, 'The office location you are leaving from does not exist');
       const tripDestination = await OfficeLocationRepository.findById({ uuid: destination });
       if (!tripDestination) return sendErrorResponse(res, 404, 'The office location you are going to does not exist');
+      // transferring control to controller methode based on request type
```
Comment (chukwuemekachm) [inline src/controllers/TripRequestController.js]: Remove this as well.

**Review by chukwuemekachm — CHANGES_REQUESTED** [review]

**Review by chukwuemekachm — APPROVED** [review]
LGTM

**Pull request merged**